src/sympc/session/session.py

```diff
@@ -128,8 +122,10 @@ def __init__(
         self.trusted_third_party = ttp
 
         # The CryptoStore is initialized at each party when it is unserialized
-        self.crypto_store: Optional[Any] = None  # TODO: this should be CryptoStore
+        self.crypto_store: Dict[Any, Any] = None  # TODO: this should be CryptoStore
```

Comment:
Q: Could you make this ```Optional[Dict[Any, Any]]``` (the Optional keyword is to "mark" the variable that it might also be ```None```)

Reply:
Done 👍